open_api_framework/conf/base.py

```diff
@@ -556,156 +565,377 @@ def connect(
     group="Celery",
 )
 
+USE_STRUCTLOG = config(
+    "USE_STRUCTLOG",
+    default=False,
+    help_text="Enable structlog-based logging instead of stdlib logging.",
+    group="Logging",
+)
+# XXX: this should be renamed to `LOG_REQUESTS` in the next major release
+ENABLE_STRUCTLOG_REQUESTS = config(
+    "ENABLE_STRUCTLOG_REQUESTS",
+    default=False,
```

Comment:
Since this only works if `USE_STRUCTLOG=True`, let's keep this on by default, since we'll usually want this if structlog is enabled
```suggestion
    default=True,
```